docs/web-renderer.1

```diff
@@ -0,0 +1,34 @@
+.TH "WEB-RENDERER" 1
+.SH NAME
+web-renderer \- present web content in a native-looking window
+.SH SYNOPSIS
+web-renderer [options]
+.SH DESCRIPTION
+Application that presents web content in a way that looks like a native application window.
+The application will start in fullscreen mode by default.
+.SH OPTIONS
+--wm, --windowed            Start application in windowed mode.
+
+--ww, --width <width>       Window width relative to screen, from 0 to 1.
+
+--wh, --height <height>     Window height relative to screen, from 0 to 1.
+
+-u, --url <url>             URL to open. Defaults to localhost.
+
+-t, --window-title <title>  Specify a title for the window.
+
+-i, --icon <path>           Specify the path to an icon to be set as the application icon.
+
+--wf, --show-window-frame   Show the frame of the window.
+
+-h, --help                  Displays a help message.
```

Comment:
```suggestion
-k, --kiosk
           Start in kiosk mode. In this mode, the application will start in fullscreen with
           minimal UI. It will prevent the user from quitting or performing any interaction
           outside of usage of the application.

           Note that it can't offer a complete secure solution by itself. Your kiosk system must
           have additional configuration and security settings to lock down the OS. In
           particular, you must configure or disable the window manager, limit the session
           capabilities, use some restart/watchdog mechanism, disable VT switching etc.

           All window-based arguments are ignored with this option.

-w, --width <width>
           Window width relative to screen, from 0 to 1.

-h, --height <height>
           Window height relative to screen, from 0 to 1.

-t, --title <title>
           Specify a title for the window.

-i, --icon <path>
           Specify the path to an icon to be set as the application icon.

-b, --show-window-border
           Show a window border. Without this, the user cannot move or resize the window
           via the window system.

-h, --help
           Displays a help message.
```

Reply:
Not 100% sure of the formatting here, but I think that this is a better API. The URL shouldn't be an, it should be an optional positional argument at the end.